index.html

```diff
@@ -230,10 +238,6 @@ <h2>Media Type Registration</h2>
                 document, where X+Y means that both X and Y processor
                 profiles must be supported, and X|Y means that either
                 X or Y processor profile must be supported.</p>
-              <p>For more information about processor profile
```

Comment:
Do we really need to remove this sentence? Doing so will mean we have to go through the review process and update the media type registration, which I'm happy to do if there's a good argument in favour, but reluctant to do otherwise.

Reply:
I don't understand the "doing so ...". Also I don't understand why the + and | are not defined in the media type registration itself.

Reply:
@nigelmegitt the removed sentence is an FYI that contains misleading information (since the semantics of the two operators are not in fact related to `ttp:processorProfileCombination`. Removing this sentence, while a literal change to the text of the registration, only serves to remove strictly informative information (that happens to be wrong). As such, I would argue that this does not require going through an IANA review process, but I'm not the one to make that call.

Reply:
Discussed briefly in today's TTWG meeting - @skynavga is of the view that the text being removed is misleading and should be removed, and does not need a full IETF expert review because it's "for information" text being removed.

Reply:
@cconcolato we are trying to avoid doing an IANA review, so changing the body of section 2 in a substantial manner, such as add new normative semantics about `[+|]`, is an undesirable approach at this time;